AllReadyApp/Web-App/AllReady/Features/Notifications/VolunteerAssignedToItineraryHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Threading.Tasks;
+using AllReady.Models;
+using MediatR;
+using Microsoft.Data.Entity;
+
+namespace AllReady.Features.Notifications
+{
+    public class VolunteerAssignedToItineraryHandler : IAsyncNotificationHandler<VolunteerAssignedToItinerary>
+    {
+        private readonly AllReadyContext _context;
+        private readonly IMediator _mediator;
+
+        public VolunteerAssignedToItineraryHandler(AllReadyContext context, IMediator mediator)
+        {
+            _context = context;
+            _mediator = mediator;
+        }
+
+        public async Task Handle(VolunteerAssignedToItinerary notification)
+        {
+            var taskSignup = await _context.TaskSignups
+                .AsNoTracking()
+                .Include(x => x.User)
+                .SingleAsync(ts => ts.Id == notification.TaskSignupId)
+                .ConfigureAwait(false);
```

Comment:
No, if @SeanFeldman told you to do it this way then you're doing it correctly. I'm just saying I don't fully understand it and need to do some research.